content/docs/1.7.0/references/settings.md

```diff
@@ -746,6 +746,23 @@ The available volume spec options are:
   Longhorn does not forcefully re-schedule the replicas to a zone that does not have enough nodes
   to support even balance. Instead, Longhorn will re-schedule to balance at the node level.
 
+#### Replica Auto Balance Disk Pressure Threshold (%)
+
+> Default: `90`
+
+Percentage of currently used storage that triggers automatic replica rebalancing.
+
+When the threshold is reached, Longhorn automatically rebuilds replicas that are under disk pressure on another disk within the same node.
+
+To disable this feature, set the value to 0.
+
+This setting takes effect only when the following conditions are met:
+
+- [Replica Auto Balance](#replica-auto-balance) is set to **best-effort**. To disable this feature when replica auto-balance is set to best-effort, set the value to 0.
```

Comment:
```suggestion
- [Replica Auto Balance](#replica-auto-balance) is set to **best-effort**. To disable this setting (disk pressure threshold) when replica auto-balance is set to best-effort, set the value of this setting to **0**.
```

Reply:
In this context, "this setting" could mean either, so I specified the setting name to avoid confusing users.